LinuxFunctionsStacks.json

```diff
@@ -28,6 +28,26 @@
                         "isPreview": false,
                         "isDeprecated": false,
                         "isHidden": false
+                    },
+                    {
+                        "displayVersion": "2.2",
+                        "runtimeVersion": "dotnet|2.2",
+                        "supportedFunctionsExtensionVersions": [
+                            "~2"
+                        ],
+                        "isDefault": true,
+                        "minorVersions": [],
+                        "applicationInsights": true,
+                        "appSettingsDictionary": {
+                            "FUNCTIONS_WORKER_RUNTIME": "dotnet"
+                        },
+                        "siteConfigPropertiesDictionary": {
+                            "Use32BitWorkerProcess": false,
+                            "linuxFxVersion": "dotnet|2.2"
+                        },
+                        "isPreview": false,
+                        "isDeprecated": false,
```

Comment:
Actually this should be true. Dotnet 2.2 was deprecated dec 23 2019

Reply:
Functions V2 is not deprecated yet. Lets sync up offline for more details on this.